web/src/components/hooks/useIsHelmManaged.js

```diff
@@ -1,57 +1,39 @@
-import { useState, useEffect } from "react";
+import { useQuery } from "react-query";
 import { Utilities } from "../../utilities/utilities";
 
 async function fetchIsHelmManaged({
   accessToken = Utilities.getToken(),
   apiEndpoint = process.env.API_ENDPOINT,
+  _fetch = fetch,
```

Comment:
I like to inject fetch into functions so I don't have to mock them globally in unit tests.